patches/tsc-multi@1.1.0.patch

```diff
@@ -269,7 +375,33 @@ index 120865b0bca77f07f51b3d3e622ee269b9a1a24c..82baa55dd386b3dc78a66055ab6ce925
  function loadCompiler(cwd, name = "typescript") {
      const path = require.resolve(name, { paths: [cwd, __dirname] });
      return require(path);
-@@ -43,31 +46,82 @@ class Worker {
+ }
++// Any paths given by typescript will be normalized to forward slashes.
++// Local paths should be normalized to make any comparisons.
++const directorySeparator = "/";
++const backslashRegExp = /\\/g;
++function normalizeSlashes(path) {
++    return path.includes("\\")
++        ? path.replace(backslashRegExp, directorySeparator)
++        : path;
++}
++function hashPackageOverrides(overrides) {
++    if (overrides === undefined)
++        return "";
++    const str = JSON.stringify(overrides);
++    if (str.length === 0)
++        return "";
++    let hash = 0;
++    for (let i = 0; i < str.length; i++) {
++        hash = (hash << 5) - hash + str.charCodeAt(i);
++        hash |= 0; // Convert to 32bit integer
++    }
++    return hash;
++}
```

Comment:
Nit: classic DBJ2 initializes 'hash = 5381':
https://stackoverflow.com/questions/10696223/reason-for-the-number-5381-in-the-djb-hash-function

And adds rather than subtracts:
http://www.cse.yorku.ca/%7Eoz/hash.html